src/PerformanceSensitiveAnalyzers/UnitTests/CallSiteImplicitAllocationAnalyzerTests.cs

```diff
@@ -149,5 +149,97 @@ public void Foo()
                 // Test0.cs(12,9): warning HAA0102: Non-overridden virtual method call on a value type adds a boxing or constrained instruction
                 VerifyCS.Diagnostic(CallSiteImplicitAllocationAnalyzer.ValueTypeNonOverridenCallRule).WithLocation(12, 9));
         }
+
+        [Fact]
+        public async Task CallSiteImplicitAllocation_ObjectParamsCalledWithStringArray_NoWarning()
+        {
+            var sampleProgram = @"
+using System.IO;
+using Roslyn.Utilities;
+
+public class MyClass
+{
+    [PerformanceSensitive(""uri"")]
+    public void Testing()
+    {
+        string[] args = new [] { ""foo"", ""bar""};
```

Comment:
Please rewrite the tests to avoid the use of "foo", "bar" words. They are disallowed by Microsoft.

@mavasani Might be worth updating the contribution doc and maybe create an analyzer for these, WDYT?